### PR TITLE
Milestone 3: Finish math app

### PR DESCRIPTION
**Pull request by mohisa302**
### Finish math app
in this Mile stone I:

- Added all the pages
- Used router
- Added styles

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Route, Routes } from 'react-router-dom';
+import Home from './components/Home';
+import Quotes from './components/Quotes';
+import Calculator from './components/Calculator';
+import './styles/App.css';
+import Header from './components/Header';
+
+/* import React, { useState } from 'react'; */
+/* <nav className={styles.nav}></nav> */
+const App = () => (
```
Comment (AmaduKamara) [inline src/App.jsx]: - [ ] Let's try not to have commented unused codes from our codebase as they make our codebase look dirty and unprofessional. Kindly remove all unused commented codes from your codebase so your code looks clean and professional 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @mohisa302,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Created 3 pages for the app as required 👍🏽
✅ The app works fine as expected and all routes work fine 👍🏽

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.jsx (hunk 1 above)
Comment (peterrobert) [reply src/App.jsx]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @mohisa302, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**